implementation/test/TBTCSystemTest.js

```diff
@@ -9,30 +9,41 @@ const {BN, expectRevert} = require("@openzeppelin/test-helpers")
 const {expect} = require("chai")
 
 const TBTCSystem = contract.fromArtifact("TBTCSystem")
+const TestTBTCDepositToken = contract.fromArtifact("TestTBTCDepositToken")
 
 describe("TBTCSystem", async function() {
   let tbtcSystem
   let ecdsaKeepFactory
+  let tdt
```

Comment:
Can we name it `tbtcDepositToken`?

Reply:
tbtcDepositToken is in use by deployAndLinkAll

Reply:
🤷‍♂